docs/content/getting-started/basics.md

```diff
@@ -86,52 +88,53 @@ class User
 }
 ```
 
-With this configuration, after you launch `console db-tools:anonymize`, all your user's email addresses
-will be gone and replace with hashed ones.
+With this configuration above, after you launched `console db-tools:anonymize`,
+all your user's email addresses will be gone and replaced with hashed ones.
```

Comment:
```suggestion
With the above configuration, after you launched `console db-tools:anonymize`,
all your user's email addresses will be gone and replaced with hashed ones.
```

I think we need to choose `this configuration` or `the above configuration` but not both ^^